src/operator/api/index.ts

```diff
@@ -0,0 +1,408 @@
+      date: '01/01/2011',
+      account: {
+        id: '345',
+        marketplace: {
+          name: 'Amazon Web Services',
+          url: 'smile.amazon.com',
+          shortName: 'aws',
+        },
+        balance: 10,
+        billingContact: {
+          companyName: 'Data',
+          email: 'watts@influxdata.com',
+          firstName: 'Andrew',
+          lastName: 'Watkins',
+          country: 'USA',
+          street1: '345 Main St',
+          city: 'Austin',
+          subdivision: 'TX',
+          postalCode: 50000,
+        },
+        users: [],
+        type: 'cancelled',
+      },
+    },
+    {
+      id: '678',
+      idpeID: '678',
+      name: 'Lucky 3',
+      region: 'gcp-west',
+      provider: 'gcm',
+      date: '01/01/2012',
+      account: {
+        id: '678',
+        marketplace: {
+          shortName: 'gcm',
+          name: 'Google Cloud Marketplace',
+          url: 'www.google.com',
+        },
+        balance: 20,
+        billingContact: {
+          companyName: 'Pineapple',
+          email: 'desa@influxdata.com',
+          firstName: 'Michael',
+          lastName: 'De Sa',
+          country: 'USA',
+          street1: '678 Main St',
+          city: 'Seattle',
+          subdivision: 'WA',
+          postalCode: 80000,
+        },
+        users: [],
+        type: 'free',
+      },
+    },
+  ]
+
+  const filtered = organizations.filter(org => {
+    if (searchTerm) {
+      return org.id.includes(searchTerm) || org.idpeID.includes(searchTerm)
+    }
+    return true
+  })
+  return makeResponse(200, filtered)
+}
+
+export const getOrgById = (_id: string): ReturnType<typeof getOrg> => {
+  const organization: Organization = {
+    id: '123',
+    idpeID: '123',
+    name: 'Best Org',
+    region: 'us-west',
+    provider: 'Zuora',
+    date: '01/01/2010',
+    account: {
+      id: '123',
+      marketplace: null,
+      balance: 0,
+      billingContact: {
+        companyName: 'Influx',
+        email: 'asalem@influxdata.com',
+        firstName: 'Ariel',
+        lastName: 'Salem',
+        country: 'USA',
+        street1: '123 Main St',
+        city: 'New York',
+        subdivision: 'NY',
+        postalCode: 30000,
+      },
+      users: [],
+      type: 'pay_as_you_go',
+    },
+  }
+
+  return makeResponse(200, organization)
+}
+
+export const getMe = (): ReturnType<typeof apiGetMe> => {
+  const me: Me = {
+    id: '123',
+    firstName: 'ariel',
+    lastName: 'salem',
+    email: 'asalem@influxdata.com',
+    orgId: 'org123',
+    accountId: 'account123',
+    isBeta: false,
+    isOperator: true,
+    marketplace: null,
+    balance: 0,
+    billingContact: {
+      companyName: 'Influx',
+      email: 'asalem@influxdata.com',
+      firstName: 'Ariel',
+      lastName: 'Salem',
+      country: 'USA',
+      street1: '123 Main St',
+      city: 'New York',
+      subdivision: 'NY',
+      postalCode: 30000,
+    },
+    users: [],
+    type: 'pay_as_you_go',
+  }
```

Comment:
This looks like a much larger payload than I expected from `/me`, but I guess that change to the swagger file was in a previous PR? Do we need all of this here?